app-information.html

```diff
@@ -0,0 +1,369 @@
+            "background_color": "red",
+            "screenshots": [{
+              "src": "screenshots/in-game-customize.jpg",
+              "sizes": "1280x920",
+              "type": "image/jpeg",
+              "label": "Screenshot of the car customization screen."
+            },{
+              "src": "screenshots/in-game-driving.jpg",
+              "sizes": "1280x920",
+              "type": "image/jpeg",
+              "label": "Screenshot of the the driving experience with Heads-up Display (HUD) active."
+            }]
+          }
+        </pre>
+      </section>
+    </section>
+    <section>
+      <h2>
+        Marketable web applications
+      </h2>
+      <p>
+        As web applications become available via more services (e.g., digital
+        storefronts, search results) where they are presented alongside native
+        applications, it becomes quite important for them to appear similarly
+        to their native counterparts. A web application can define both its
+        {{WebAppManifest/name}} and {{WebAppManifest/icons}} in the manifest,
+        but those keys do not provide enough information to help users
+        determine whether they might want to install it. Users have become
+        accustomed to being offered more details about the applications they
+        consider installing, and this document defines additional manifest
+        members that can provide that.
+      </p>
+    </section>
+    <section id="processing" data-link-for="WebAppManifestAppInfo">
+      <h3>
+        Extension to processing the manifest
+      </h3>
+      <p>
+        The following steps for <dfn>processing application info in a
+        manifest</dfn> are run during the extension point in[=processing the
+        manifest=].
+      </p>
+      <ol>
+        <li>Set <var>manifest</var>["<a>categories</a>"] to the result of
+        running <a>processing the <code>categories</code> member</a> given
+        <var>manifest</var>["<a>categories</a>"].
+        </li>
+        <li>Set <var>manifest</var>["<a>screenshots</a>"] to the result of
+        running <a>processing `ManifestImageResource` members</a> given
+        <var>manifest</var>["<a>screenshots</a>"] and <var>manifest URL</var>.
+        </li>
+      </ol>
+      <p class="issue">
+        The {{WebAppManifestAppInfo/description}} and
+        {{WebAppManifestAppInfo/iarc_rating_id}} would be set when converting
+        the manifest json to a WebAppManifest distionary.
+      </p>
+    </section>
+    <section data-dfn-for="WebAppManifestAppInfo" data-link-for=
+    "WebAppManifestAppInfo">
+      <h2>
+        <dfn data-dfn-for="">WebAppManifest</dfn> dictionary
+      </h2>
+      <pre class="idl">
+          partial dictionary WebAppManifest {
+            sequence&lt;USVString&gt; categories;
+            USVString description;
+            DOMString iarc_rating_id;
+            sequence&lt;ManifestImageResource&gt; screenshots;
```

Comment:
I guess that makes this a bit less informative if we change it to `"screenshots"` - like we loose all the type information, which can be quite useful